camel/loaders/chunkr_reader.py

```diff
@@ -0,0 +1,152 @@
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+import json
+import logging
+import os
+import time
+from typing import IO, Any, Optional, Union
+
+import requests
+
+from camel.utils import api_keys_required
+
+CHUNKR_ENDPOINT = "https://api.chunkr.ai/api/v1/task"
+
+logging.basicConfig(level=logging.INFO)
+
+
+class ChunkrReader:
+    r"""Chunkr Reader for processing documents and returning content
+    in various formats.
+
+    Args:
+        api_key (Optional[str], optional): The API key for Chunkr API. If not
+            provided, it will be retrieved from the environment variable
+            `CHUNKR_API_KEY`. Defaults to None.
+        timeout (int, optional): The maximum time in seconds to wait for the
+            API responses. Defaults to 30.
+        **kwargs (Any): Additional keyword arguments for request headers.
+    """
+
+    @api_keys_required("CHUNKR_API_KEY")
+    def __init__(
+        self,
+        api_key: Optional[str] = None,
+        timeout: int = 30,
+        **kwargs: Any,
+    ) -> None:
+        api_key = self._api_key = api_key or os.getenv('CHUNKR_API_KEY')
```

Comment:
```suggestion
        self._api_key = api_key or os.getenv('CHUNKR_API_KEY')
```

Reply:
Thanks, @Wendong-Fan ! Good catch. I’ll go ahead and make that update.